index.html

```diff
@@ -19,25 +18,27 @@ <h1>Welcome to Tic Tac Toe</h1>
       <table>
         <tr>
           <!-- the onclick calls a function called "handleClick" and passes itself to it -->
-          <td id='top-left' onclick="handleClick(this)" ></td>
-          <td id='top-middle' onclick="handleClick(this)"></td>
-          <td id='top-right' onclick="handleClick(this)"></td>
+          <td onclick="handleClick(this)" id="0-0"></td>
```

Comment:
ideally the id attribute should be placed before any other attribute. 
`<td id="0-0" onclick="handleClick(this)"></td>`